control/iosys.py

```diff
@@ -2203,24 +2203,21 @@ def linearize(sys, xeq, ueq=None, t=0, params=None, **kw):
     copy_names : bool, Optional
         If True, Copy the names of the input signals, output signals, and
         states to the linearized system.
-
-    Returns
-    -------
-    ss_sys : LinearIOSystem
-        The linearization of the system, as a :class:`~control.LinearIOSystem`
-        object (which is also a :class:`~control.StateSpace` object.
-
-    Additional Parameters
-    ---------------------
```

Comment:
As above, this should be named "Other Parameters" and should go _after_ the "Returns" section.